demo/dev/index.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
+import VueProjectionGrid from '../../src/index.vue';
+
```

Comment:
The same as that for React, not necessary to duplicate the demo code, you can use webpack resolve config to differentiate the source code reference.